pkg/receiver/gitproviderreceiver/internal/scraper/githubscraper/genqlient.graphql

```diff
@@ -1,3 +1,7 @@
+# This query currently costs 1 point per repo. REEVALUATE THIS ESTIMATE if you
+# change this query. And please update our docs if you do!
+# docs: pkg/receiver/gitproviderreceiver/README.md#rate-limiting
```

Comment:
I think this needs to be the full URL since this is a graphql file and not markdown so might not auto render.